swift/apple/FirezoneNetworkExtension/CallbackHandler.swift

```diff
@@ -20,7 +20,8 @@ public protocol CallbackHandlerDelegate: AnyObject {
   func onSetInterfaceConfig(
     tunnelAddressIPv4: String,
     tunnelAddressIPv6: String,
-    dnsAddresses: [String]
+    dnsServers: [String],
+    searchDomains: [String],
```

Comment:
```suggestion
    searchDomains: [String]
```

Reply:
I am pretty sure I had this and it complained but I'll try again.

Reply:
Ha! It suddenly builds. Might have been an old cache or something.